gulp/build.js

```diff
@@ -85,7 +85,7 @@ function buildStylesWrapper(options) {
   return function buildStyles() {
     const sassCompiler = sass({
       includePaths: [
-        path.join(process.cwd(), 'node_modules')
+        path.join(process.cwd(), 'libs')
```

Comment:
Het enige wat we importeren in sass is `bootstrap-sass`, en die staat vanaf nu in `libs`